app/views/petitions/moderation_info.html.erb

```diff
@@ -0,0 +1,9 @@
+<% content_for(:page_title) { "#{@petition.title} - has been already sent to moderation - e-petitions" } %>
+<% content_for :title do %>
+<div class="title_block">
+  <h1>Thank you for your support, but <%= @petition.title %> petition has been already sent to moderation.</h1>
```

Comment:
Can we change the words to:

"<name of creator>'s petition "<petition title>" already has its 5 sponsors and we're taking a look to make sure it meets the Petitions service terms and conditions."

"The petition should be available for signing in a few days".